src/common/transformations/src/transformations/common_optimizations/select_with_one_value_condition.cpp

```diff
@@ -84,14 +82,9 @@ ov::pass::SelectWithOneValueCondition::SelectWithOneValueCondition() {
 
             auto target_shape = copy_to.make<Constant>(element::i32, Shape{select_rank}, select_shape_values);
             auto broadcast = copy_to.make<Broadcast>(branch_output, target_shape);
-            select->output(0).replace(broadcast->output(0));
-            broadcast->set_friendly_name(select->get_friendly_name());
-            copy_runtime_info(select, copy_to.get());
-        } else {
-            return false;
+            return replace_output_update_name(select->output(0), broadcast->output(0));
```

Comment:
in the first branch (`if (select_shape.is_static() && branch_output_shape.same_scheme(select_shape)) {`), the fix is correct, but here
I think we can save friendly name here because `broadcast` is a new node in the graph after Parameter.
We also need to copy rt_info to `target_shape` constant,
so I suggest to revert the changes.

Reply:
Right, it's not needed for the fix.